tests/.editorconfig

```diff
@@ -3,8 +3,10 @@ dotnet_diagnostic.CA1003.severity = none # Use generic event handler instances
 dotnet_diagnostic.CA1034.severity = none # Do not nest type
 dotnet_diagnostic.CA1051.severity = none # Do not declare visible instance fields
 dotnet_diagnostic.CA1062.severity = none # In externally visible method, validate parameter is non-null before using it
+dotnet_diagnostic.CA1510.severity = none # Use 'ArgumentNullException.ThrowIfNull' instead of explicitly throwing a new exception instance - remove once we drop LACKS_THROWIFNULL

```

Comment:
Many instances in the tests, and I ran into trouble making a convenience method to access throughout. Can try again if you like.